frappe/utils/image.py

```diff
@@ -17,3 +17,26 @@ def resize_images(path, maxdim=700):
 					im.save(os.path.join(basepath, fname))
 
 					print("resized {0}".format(os.path.join(basepath, fname)))
+
+def strip_exif_data(content, content_type):
+	""" Strips exif from image files which support it.
+
+	Works by creating a new Image object which ignores exif by
+	default and then extracts the binary data back into content.
+
+	Returns: stripped image content
+	"""
+
+	from PIL import Image
+	import io
+
+	original_image = Image.open(io.BytesIO(content))
+	output = io.BytesIO()
+	
+	new_image = Image.new(original_image.mode, original_image.size)
```

Comment:
Does image.new performs exif stripping work?

Reply:
Yes, when new image is created, old EXIF data is stripped.
There is no method in PIL to explicitly strip EXIF.